.github/workflows/unit-integration-tests.yml

```diff
@@ -30,9 +30,11 @@ jobs:
         with:
           repository: 'DataDog/dd-trace-go'
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
+        uses: reviewdog/action-golangci-lint@v2
```

Comment:
Just double checking with you, my understanding is that `reviewdog/action-golangci-lint` is a wrapper for `golangci-lint` so the instruction to run the linter locally documented [here](https://github.com/DataDog/dd-trace-go/blob/main/CONTRIBUTING.md#style-guidelines) still applies and we don't need to update it after this change, correct?

Reply:
correct! `reviewdog` just runs `golangci-lint` and parses the results to post code review comments, so the linting setup doesn't change at all